### ISSUE TITLE
Whitespace shouldn't be blocked as long we use "Return" to select tags from the dropdown list

### ISSUE THREAD

**Issue by thiagocamposviana** [ezsystems/eztags#107]
Hi

There is one issue at the moment where whitespace is blocked, so the user can type whitespace in order to further filter tags with whitespace. The whitespace was supposed to be used to select a focused tag but right now that is what "Return" is doing.

As a workaround it is necessary for the user to press "Esc" and then type whitespace.

This is easily solvable with this patch:
 ```
diff --git a/ezpublish_legacy/extension/eztags/design/standard/javascript/jquery.eztags.js b/ezpublish_legacy/extension
index 2573d71..100db3d 100644
--- a/ezpublish_legacy/extension/eztags/design/standard/javascript/jquery.eztags.js
+++ b/ezpublish_legacy/extension/eztags/design/standard/javascript/jquery.eztags.js
@@ -56,9 +56,8 @@
       LEFT: 37,
       UP: 38,
       RIGHT: 39,
-      DOWN: 40,
-      SPACE: 32
-    },
+      DOWN: 40
+  },
 
     is_key: function(e, keys){
       var self = this;
@@ -563,7 +562,7 @@
   Base.prototype.navigate_autocomplete_dropdown = function(e) {
     if(EzTags.is_key(e, 'ESC')){ this.close_autocomplete();}
 
-    if(!EzTags.is_key(e, ['UP', 'DOWN', 'SPACE'])){return;}
+    if(!EzTags.is_key(e, ['UP', 'DOWN'])){return;}
     var $items = this.$autocomplete_tags.find('a');
 
     if (!$items.length){ return;}
@@ -573,9 +572,7 @@
     e.stopPropagation();
 
     var index = $items.index(e.target);
-    if(e.which === 'SPACE' && index >= 0){
-      $items.eq(index).trigger('click');
-    }
+
 
     EzTags.is_key(e, 'UP') && index >= 0                   && index--;
     EzTags.is_key(e, 'DOWN') && index < $items.length - 1  && index++;

**Comment by emodric** [comment]
Hi! Thanks for the issue report! I'll take a look at it in coming days.

**Comment by emodric** [comment]
Fixed in d4139f7773685fe8981672f33f48f77efcb046ea

Thanks @thiagocamposviana !
